components/my-account/Lists/ListTable/ListTable.tsx

```diff
@@ -0,0 +1,171 @@
+                <TableCell style={{ ...styles.tableCellStyles, width: mdScreen ? '25%' : '10%' }}>
+                  {mdScreen ? (
+                    <Box style={{ justifyContent: 'flex-end', display: 'flex' }}>
+                      <Button sx={styles.iconButtonStyles} data-testid="initiateQuoteBtn">
+                        {t('initiate-quote')}
+                      </Button>
+                      <Button sx={styles.iconButtonStyles} data-testid="addToCartBtn">
+                        {t('add-to-cart')}
+                      </Button>
+                      <IconButton
+                        sx={styles.iconButtonStyles}
+                        onClick={() => onEditList(item.id)}
+                        data-testid="editBtn"
+                      >
+                        <Edit />
+                      </IconButton>
+                      <IconButton
+                        sx={styles.iconButtonStyles}
+                        onClick={() => onCopyList(item.id)}
+                        data-testid="copyBtn"
+                      >
+                        <FolderCopySharp />
+                      </IconButton>
+                      <IconButton
+                        sx={styles.iconButtonStyles}
+                        onClick={() => onDeleteList(item.id)}
+                        data-testid="deleteBtn"
+                      >
+                        <Delete />
+                      </IconButton>
+                    </Box>
+                  ) : (
+                    <>
+                      <IconButton
+                        style={{ padding: '0px' }}
+                        onClick={(e) => setAnchorEL(e.currentTarget)}
+                        data-testid="menuBtn"
+                      >
+                        <MoreVert />
+                      </IconButton>
+                      <Menu
+                        anchorEl={anchorEl}
+                        open={Boolean(anchorEl)}
+                        onClose={() => setAnchorEL(null)}
+                      >
+                        {options.map((option, i) => (
+                          <MenuItem
+                            key={option.name}
+                            onClick={() => {
+                              option.onClick(item.id)
+                              setAnchorEL(null)
+                            }}
+                            style={
+                              i !== options.length - 1
+                                ? { borderBottom: '0.5px solid #EAEAEA' }
+                                : {}
+                            }
+                          >
+                            {option.name}
+                          </MenuItem>
+                        ))}
+                      </Menu>
+                    </>
+                  )}
+                </TableCell>
+              </TableRow>
+            )
+          })}
+        </TableBody>
+      </Table>
+    </TableContainer>
+  )
+}
+
+export default ListTable
```

Comment:
**Using correct types**
```
interface ListTableProps {
  rows: CrWishlist[]
  onDeleteList: (param: Maybe<string> | undefined) => void
  onCopyList: (param: Maybe<string> | undefined) => void
  onEditList: (param: Maybe<string> | undefined) => void
  isLoading: boolean
}
```

should be
```
interface ListTableProps {
  rows: CrWishlist[]
  onDeleteList: (param: string) => void
  onCopyList: (param: string) => void
  onEditList: (param: string) => void
  isLoading: boolean
}
```

**Use Translation instead of hardcoding**
```
const options = [
    { name: 'Edit', onClick: onEditList },
    { name: 'Add list items to cart', onClick: () => alert('Work in progress') },
    { name: 'Initiate Quote', onClick: () => alert('Work in progress') },
    { name: 'Duplicate', onClick: onCopyList },
    { name: 'Delete', onClick: onDeleteList },
  ]
```

Reply:
done